rubygem-patternfly-sass/rubygem-patternfly-sass.spec

```diff
@@ -0,0 +1,81 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name patternfly-sass
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.8.0
+Release: 1%{?dist}
+Summary: Red Hat's Patternfly, converted to Sass and ready to drop into Rails
+Group: Development/Languages
+License: Apache-2.0
+URL: https://github.com/Patternfly/patternfly-sass
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildArch: noarch
+
+%description
+Red Hat's Patternfly, converted to Sass and ready to drop into Rails.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+mkdir -p .%{gem_dir}
+%{?scl:scl enable %{scl} "}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0}
+%{?scl:"}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+
+
+# Run the test suite
+%check
```

Comment:
ideally needs filling in to run whatever specs are in the project, else it should be removed